src/config/apigee.googleapis.com_ProxyV2.yml

```diff
@@ -21,8 +21,14 @@ gcp:
     id: env
   - value: resource.labels.location
     id: location
+    compatibilityModeOnly: true
+  - value: resource.labels.location
+    id: gcp.region
   - value: resource.labels.proxy_name
     id: proxy_name
+    compatibilityModeOnly: true
+  - value: resource.labels.proxy_name
+    id: gcp.instance.name
   - value: resource.labels.runtime_version
     id: runtime_version
   - value: resource.labels.instance_id
```

Comment:
Shouldn't we create new dimension `gcp.instance.id` here too? or maybe we have already created such dimension in the code (couldn't find). @pgutkowski could you maybe confirm?

Reply:
After discussion with both Maria & @pawelsiwek, I'm also adding those fields that were there in odin spec here:
https://bitbucket.lab.dynatrace.org/projects/ODIN/repos/dt-semantic-conventions/browse/doc/semconv/cloud_gcp.md
![image](https://user-images.githubusercontent.com/50921490/132845542-656aa1ab-0a40-469b-83c3-8d02356f78eb.png)
but not present in jira when it was created:
* `gcp.instance.id` in configs whenever applicable
* `type` equal to service name & hardcoded in code (not configs)

Actual changes still need some confirmation with Pawel on Monday 😢, as I have some discrepancies with my choice of fields for .id and .name and his in one wiki he has sent to me.